advanced/am-pattern-2/README.md

```diff
@@ -39,16 +43,19 @@
 ### Install Chart From [WSO2 Helm Chart Repository](https://hub.helm.sh/charts/wso2)
 
 ##### 1. Deploy Helm chart for WSO2 APIM Pattern 2 deployment
+
 [Option 1] Deploy using Docker images from DockerHub.
 
 ```
 helm install --name <RELEASE_NAME> wso2/am-pattern-2 --version 3.1.0-2 --namespace <NAMESPACE>
 ```
 
 [Option 2] Deploy WSO2  API Manager using Docker images from WSO2 Private Docker Registry.
+
 ```
 helm install --name <RELEASE_NAME> wso2/am-pattern-2 --version 3.1.0-2 --namespace <NAMESPACE> --set wso2.subscription.username=<SUBSCRIPTION_USERNAME> --set wso2.subscription.password=<SUBSCRIPTION_PASSWORD>
 ```
+
 **Note:**
 
 * `NAMESPACE` should be the Kubernetes Namespace in which the resources are deployed.
```

Comment:
if we can take the bullet and bring the line after the ':' better
there is another instance of the same few lines down

Reply:
Can make the status to Resolve